README.md

```diff
@@ -201,3 +201,58 @@ AWS OpsWorks Chef Deployment
   ```ruby
   include_recipe 'datadog::dd-agent'
   ```
+
+Resources
+=========
+
+datadog_monitor
+---------------
+
+The `datadog_monitor` resource will help you to enable Agent integrations.
+
+The default action `:add` of this resource will automatically enable the integration by filling values in the configuration template file and set the correct permissions after having written the configuration on the filesystem. Also, it will take care of restarting the Agent.
+
+The `:remove` action disables an integration from a node.
+
+
+### Syntax
+
+```ruby
+datadog_monitor 'name' do
+  init_config                       Hash, default: {}
+  instances                         Array, default: []
+  logs                              Array, default: []
+  use_integration_template          true, false, default: false
+  action                            Symbol, default: :add
+end
+```
+
+#### Actions
+
+There is two available actions in this resource:
```

Comment:
I don't think this sentence is necessary :)

(for me, the shorter the docs are the better)